src/main/client/src/components/@commons/ImageCarousel.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { Box, Button, ButtonGroup, Flex, Image } from '@chakra-ui/react'
+import { ChevronLeftIcon, ChevronRightIcon } from '@chakra-ui/icons'
+
+type ImageCarouselProps = {
+  images: string[]
+}
+
+export const ImageCarousel: React.FC<ImageCarouselProps> = ({ images }) => {
+  const [currentImageIndex, setCurrentImageIndex] = useState<number>(0)
+  const previousImage = () => {
+    const previousIndex = currentImageIndex - 1
+    setCurrentImageIndex(previousIndex < 0 ? images.length - 1 : previousIndex)
+  }
+  const nextImage = () => {
+    const nextIndex = currentImageIndex + 1
+    setCurrentImageIndex(nextIndex > images.length - 1 ? 0 : nextIndex)
+  }
+  if (images.length === 0) return null
+  return (
```

Comment:
Maybe with `return (images.length === 0) && ...`

Reply:
I don't think that is pussible. A boolean would be returned IMO, and it is not readable. Also you won't earn anything from that.

Reply:
Yeah, boolean would be returned, but: https://stackoverflow.com/questions/64019668/what-type-should-be-used-on-a-react-component-that-returns-a-boolean-or-jsx